Data/Sector/Hong-Kong-Sector-File.sct

```diff
@@ -6254,5748 +6138,368 @@ TRAN/TAMOT->SIKOU         N000.00.00.000 E000.00.00.000 N000.00.00.000 E000.00.0
-                                         N022.03.51.000 E114.55.25.920 N021.46.00.000 E114.55.25.920 TWR&CTR
-                                         N021.46.00.000 E114.55.25.920 N021.40.27.400 E114.41.57.470 TWR&CTR
-                                         N021.40.27.400 E114.41.57.470 N021.40.27.400 E114.22.00.000 TWR&CTR
-                                         N021.40.27.400 E114.22.00.000 N022.02.50.000 E114.14.39.715 TWR&CTR
-                                         N022.02.50.000 E114.02.30.000 N022.12.51.000 E114.08.15.000 TWR&CTR
-                                         N022.12.51.000 E114.08.15.000 N022.14.00.000 E114.11.00.000 TWR&CTR
-                                         N022.02.50.000 E114.02.30.000 N022.02.50.000 E114.14.39.715 TWR&CTR
-                                         N022.14.00.000 E114.11.00.000 N022.14.04.776 E114.11.11.759 TWR&CTR
-                                         N022.14.09.564 E114.11.23.639 N022.14.14.352 E114.11.35.520 TWR&CTR
-                                         N022.14.19.139 E114.11.47.400 N022.14.23.927 E114.11.59.279 TWR&CTR
-                                         N022.14.28.715 E114.12.11.160 N022.14.33.504 E114.12.23.040 TWR&CTR
-                                         N022.14.38.292 E114.12.34.919 N022.14.43.080 E114.12.46.799 TWR&CTR
-                                         N022.14.47.867 E114.12.58.680 N022.14.52.655 E114.13.10.559 TWR&CTR
-                                         N022.14.57.443 E114.13.22.439 N022.15.02.232 E114.13.34.320 TWR&CTR
-                                         N022.15.07.020 E114.13.46.200 N022.15.11.808 E114.13.58.079 TWR&CTR
-                                         N022.15.16.595 E114.14.09.959 N022.15.21.383 E114.14.21.840 TWR&CTR
-                                         N022.15.26.171 E114.14.33.719 N022.15.30.960 E114.14.45.599 TWR&CTR
-                                         N022.15.35.748 E114.14.57.480 N022.15.40.536 E114.15.09.360 TWR&CTR
-                                         N022.15.45.323 E114.15.21.239 N022.15.50.111 E114.15.33.120 TWR&CTR
-                                         N022.15.54.900 E114.15.45.000 N022.15.59.688 E114.15.56.879 TWR&CTR
-                                         N022.16.04.476 E114.16.08.759 N022.16.09.263 E114.16.20.640 TWR&CTR
-                                         N022.16.14.051 E114.16.32.520 N022.16.18.839 E114.16.44.399 TWR&CTR
-                                         N022.16.23.628 E114.16.56.280 N022.16.28.416 E114.17.08.160 TWR&CTR
-                                         N022.16.33.204 E114.17.20.039 N022.16.37.991 E114.17.31.919 TWR&CTR
-                                         N022.16.42.779 E114.17.43.800 N022.16.47.567 E114.17.55.679 TWR&CTR
-                                         N022.16.52.356 E114.18.07.559 N022.16.57.144 E114.18.19.440 TWR&CTR
-                                         N022.17.01.932 E114.18.31.320 N022.17.06.719 E114.18.43.199 TWR&CTR
-              
```

Comment:
I see you've drawn these intentionally to avoid having any lines overlap, but results in it being difficult to toggle certain boundaries on and off if controllers desire to. For example if for whatever reason I want to create a TMA asr just for controlling the TME sector, I can't, because it looks like this:
![image](https://github.com/vatsimhk/Hong-Kong-Sector-Package/assets/55276992/746de49e-9a83-431b-830f-4a84907c514b)

I'd recommend either adding all of the extra lines back in so each sector is "complete" when displayed. If the overlapping lines looks too messy (which they shouldn't if they are exactly the same?) then we could make a "ALL BOUNDARIES" GEO that contains all of this version, which is displayed by default. Then have each of the individual sectors drawn in full separately available for toggle as necessary.

Reply:
Will add the missing lines in